packages/eslint-config-spruce/index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 		'no-undef': 'error',
 		'no-var': 'error',
 		'no-unreachable': 'error',
-		'no-unused-vars': 'error',
+		'no-unused-vars': 'warn',
```

Comment:
Can you just comment vars out if you don't actually want to use them? If this is for a specific project (like spruce-skill) where that is onerous you can just disable it per-repo in that project's eslintrc. I'd like the default to be harsher.

Reply:
To expand on that, I see more dead vars than I do "example vars" in code. Enforcing this has been a benefit to the projects I added it for, since it 1) exposed references that had been forgotten about during our many refactors and 2) prevented people from introducing new unnecessary code during their active dev. 

I do think I see the benefit to skipping this rule in `spruce-skill`, but as a counterpoint, when I went through that repo to apply this rule I found that probably 50% of the unused vars weren't example code, they were refs that weren't doing anything in that file / weren't helpful to the end-user eng. The remaining 50% I typically commented out. I think this forces us to consider what is and isn't important as an example in these more "template"-like projects.

That said though, I'm not currently active in maintaining that project and if it feels onerous to maintain the commented-out variables I'm fine disabling it there. In other projects, I'd like to maintain the rule as-is.

Reply:
@gcpantazis Updated this to make it `error` again.

I have an override in `spruce-skill` that allows just `ctx` to be unused